x/or/rollup/client/cli/tx.go

```diff
@@ -0,0 +1,215 @@
+	"github.com/spf13/cobra"
+
+	cryptotypes "github.com/Finschia/finschia-sdk/crypto/types"
+)
+
+const (
+	FlagAmount = "amount"
+)
+
+// GetTxCmd returns the transaction commands for this module
+func GetTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      fmt.Sprintf("%s transactions subcommands", types.ModuleName),
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(NewCreateRollupCmd())
+	cmd.AddCommand(NewRegisterSequencerCmd())
+	cmd.AddCommand(NewDepositCmd())
+	cmd.AddCommand(NewWithdrawCmd())
+	// cmd.AddCommand(NewRemoveSequencerCmd())
+
+	return cmd
+}
+
+func NewCreateRollupCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		// TODO: If rollup:sequencer=1:N, add [max-sequencers] parameter
+		Use:   "create-rollup [rollup_name] [permissioned-addresses]",
+		Short: "Create a new rollup",
+		Args:  cobra.RangeArgs(1, 2),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			argRollupName := args[0]
+			argPermissionedAddresses := new(types.Sequencers)
+			if len(args) == 2 {
+				err = json.Unmarshal([]byte(args[1]), argPermissionedAddresses)
+				if err != nil {
+					return err
+				}
+			}
+
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgCreateRollup(
+				argRollupName,
+				clientCtx.GetFromAddress().String(),
+				argPermissionedAddresses,
+				// TODO: If rollup:sequencer=1:N, fix maxSequencers
+				1,
+			)
+
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func NewRegisterSequencerCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "register-sequencer [rollup_name] [pubkey]",
+		Short: "Register a new sequencer for a rollup",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			argRollupName := args[0]
+			argPubkey := args[1]
+
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			var pk cryptotypes.PubKey
+			if err := clientCtx.Codec.UnmarshalInterfaceJSON([]byte(argPubkey), &pk); err != nil {
+				return err
+			}
+
+			fAmount, _ := cmd.Flags().GetString(FlagAmount)
+			amount, err := sdk.ParseCoinNormalized(fAmount)
+			if err != nil {
+				return err
+			}
+
+			msg, err := types.NewMsgRegisterSequencer(
+				pk,
+				argRollupName,
+				clientCtx.GetFromAddress().String(),
+				amount,
+			)
+			if err != nil {
+				return err
+			}
+
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().AddFlagSet(FlagSetAmount())
+	flags.AddTxFlagsToCmd(cmd)
+
+	_ = cmd.MarkFlagRequired(FlagAmount)
```

Comment:
Why is amount is the flag? How about add it to arguments?

Reply:
That was totally my preference, but I will modify the parameters to match the bank module etc.